### ISSUE TITLE
Facebook wall post dialogue is a blank white page after logging in

### ISSUE THREAD

**Issue by taberrr** [ideashower/ShareKit#206]
Hi, noticing some weird SHKFacebook behavior... I've logged into facebook, but after logging in, immediately a white blank dialogue box is shown, where the "Post to your wall" form used to be. It was working a couple of days ago... when I tried it earlier today though, the "Connect to Facebook" window titlebar disappears and I'm left with the white blank page. Screenshot: http://cl.ly/2Z3D3R1f150Z3Z2b2v2F

Deleting my app and re-building it seemed to work once, but I tried also using [SHK logoutOfService:@"SHKFacebook"]; and that didn't work just now. I thought it used to refresh the auth token automatically.

Has anyone else seen this or know of a workaround? Thanks.

It looks like they changed their login form page: http://cl.ly/1E3N0B3Z2v3X1c0z3b3z - maybe that's related to this issue.

If you tap "Try the alternative login" it pops open Mobile Safari, yikes.

**Comment by tobytaro** [comment]
Me too, i can login ,but the share form didn't show up. It said can't find the page.
How to solve it?

**Comment by taberrr** [comment]
Since I don't yet have logout functionality built into my app, the way I got around it was just by deleting my app from my phone and re-installing it.

**Comment by adilsaleem** [comment]
Im also getting the same issue, page goes blank after login and the permissions page is no longer available

for all subsequent attempts, all i get is a blank page unless I logout from Facebook

**Comment by taberrr** [comment]
Yeah, apparently logging out and logging back in does not fix it. I just tried to test with [SHK logoutOfService:@"SHKFacebook"];

Also, closed the issue on accident, oops.

**Comment by taberrr** [comment]
Using https://github.com/ideashower/ShareKit/pull/158 I was able to fix this issue. If you want to customize your share dialogue, similar to the "old" wall post dialogue take a look at: http://developers.facebook.com/docs/reference/dialogs/feed/ and call [self.facebook dialog:@"feed" andParams:params andDelegate:self]; in your SHKFacebook's -(void) share method.

**Comment by orff** [comment]
As far as I can tell this issue is the FBconnect code in SK is out of sync with newer FB apps.  I saw this issue on a new FB app, but it worked fine on an older FB app.  For now, I am just using the older FB app as the new one is used for our web site in production and cannot be reconfigured to work with the older FB client embedded in the SK.

Hope this helps shed some light on possible fixes ( updating FBConnect in SK ?? )

**Comment by orff** [comment]
update on my previous post.  lookie here:  https://github.com/ideashower/ShareKit/pull/158

using this fork, I was able to connect to a new FB app and sharing works again
